charts/portworx/templates/portworx-operator.yaml

```diff
@@ -5,24 +5,6 @@ metadata:
   name: portworx-operator
   namespace: kube-system
 ---
-apiVersion: policy/v1beta1
-kind: PodSecurityPolicy
-metadata:
-  name: px-operator
-spec:
-  allowPrivilegeEscalation: false
-  readOnlyRootFilesystem: false
-  volumes:
-    - secret
-  runAsUser:
-    rule: 'RunAsAny'
-  seLinux:
-    rule: 'RunAsAny'
-  supplementalGroups:
-    rule: 'RunAsAny'
-  fsGroup:
-    rule: 'RunAsAny'
----
```

Comment:
PSP is not supported for `v1.25` and up, are you sure just removing all together from here is a good idea?

Reply:
@nikolaypopov I have added a condition to check and take care of this now. I have also attached the Dry-run manifest for both 1.24 and 1.25 kubernetes version for reference
[Manifest-1.24.txt](https://github.com/portworx/helm/files/10921402/Manifest-1.24.txt)
[Manifest-1.25.txt](https://github.com/portworx/helm/files/10921404/Manifest-1.25.txt)